chapter_generative-adversarial-networks/dcgan.md

```diff
@@ -5,19 +5,28 @@ In :numref:`sec_basic_gan`, we introduced the basic ideas behind how GANs work.
 
 In this section, we will demonstrate how you can use GANs to generate photorealistic images. We will be basing our models on the deep convolutional GANs (DCGAN) introduced in :cite:`Radford.Metz.Chintala.2015`. We will borrow the convolutional architecture that have proven so successful for discriminative computer vision problems and show how via GANs, they can be leveraged to generate photorealistic images.
 
-```{.python .input  n=1}
+```{.python .input}
 from mxnet import gluon, init, np, npx
 from mxnet.gluon import nn
 from d2l import mxnet as d2l
 
 npx.set_np()
 ```
 
+```{.python .input}
+#@tab pytorch
+from d2l import torch as d2l
+import torch
+import torchvision
+from torch import nn
+import warnings
```

Comment:
We don't filter out warnings. You need to solve these warnings.

Reply:
See my comments below.